apiserver/facades/agent/upgradesteps/upgradesteps.go

```diff
@@ -114,3 +182,23 @@ func (api *UpgradeStepsAPI) getMachine(canAccess common.AuthFunc, tag names.Mach
 	}
 	return machine, nil
 }
+
+func (api *UpgradeStepsAPI) getUnit(canAccess common.AuthFunc, tag names.UnitTag) (Unit, error) {
+	if !canAccess(tag) {
```

Comment:
Just curious: is it typical for upgrade steps to log at critical?

Reply:
nope, removed